chapters/data-collection.tex

```diff
@@ -1,362 +1,251 @@
-can alert you and the team that changes need to be made
-to portions of the survey.
-\texttt{bcstats} is a Stata command for back checks
-that takes the different question types into account when comparing surveys.
-
-When all data collection is complete,
-the survey team should have a final field report
-ready for validation against the sample frame and the dataset.
-This should contain all the observations that were completed;
-it should merge perfectly with the received dataset;
-and it should report reasons for any observations not collected.
-Identification and reporting of \textbf{missing data} and \textbf{attrition} is critical
-to the interpretation of any survey dataset.
-It is important to structure this reporting in a way that
-not only group broads rationales into specific categories
-but also collects all the detailed, open-ended responses to questions the field team can provide
-for any observations that they were unable to complete.
-This reporting should be validated and saved
-alongside the final raw data, and treated the same way.
-This information should be stored as a dataset in its own right
--- a \textbf{tracking dataset} --
-that records all events in which survey substitutions
-and loss to follow-up occurred in the field
-and how they were implemented and resolved.
+provides a workflow for collaborating on the resolution of duplicate entries between you and the field team.
+Next, observed units in the data must be validated against the expected sample:
+this is as straightforward as \texttt{merging} the sample list with the survey data and checking for mismatches.
+Reporting errors and duplicate observations in real-time allows the field team to make corrections efficiently.
+Tracking survey progress is important for monitoring attrition, so that it is clear early on if a change in protocols or additional tracking will be needed.
+It is also important to check interview completion rate and sample compliance by surveyor and survey team, to identify any under-performing individuals or teams.
+
+When all data collection is complete, the survey team should prepare a final field report, which should report reasons for any deviations between the original sample and the dataset collected.
+Identification and reporting of \textbf{missing data} and \textbf{attrition} is critical to the interpretation of survey data.
+It is important to structure this reporting in a way that not only group broads rationales into specific categories
+but also collects all the detailed, open-ended responses to questions the field team can provide for any observations that they were unable to complete.
+This reporting should be validated and saved alongside the final raw data, and treated the same way.
+This information should be stored as a dataset in its own right -- a \textbf{tracking dataset} -- that records all events in which survey substitutions
+and loss to follow-up occurred in the field and how they were implemented and resolved.
```

Comment:
**loss to follow-up** is written **loss-to-follow-up** in previous chapters see f3663661eef9124a3a3f4e5641ae275895d03b47. Which one should we use?

Reply:
I prefer loss to follow-up. @bbdaniels ?